cfgov/jinja2/v1/_includes/organisms/mortgage-map.html

```diff
@@ -410,8 +410,10 @@ <h3 id="mp-map-title">Percentage of mortgages {% if time_frame == '90' %} at lea
             </g>
         </svg>
     </div>
-    <p class="m-chart-footnote block__sub block__border-top block short-desc" style="font-size:14px !important"><strong>Source:</strong> Download CSV files with <a class="icon-link icon-link__download" href="{{ state_meta['url'] }}" <span class="icon-link_text">state</span></a> ({{ state_meta['size'] }}), <a class="icon-link icon-link__download" href="{{ metro_meta['url'] }}" <span class="icon-link_text">metro area</span></a> ({{ metro_meta['size'] }}), or <a class="icon-link icon-link__download" href="{{ county_meta['url'] }}"><span class="icon-link_text">county</span></a> ({{ county_meta['size'] }}) data.<br>
-        <strong>Date published:</strong> {{ pub_date_formatted }}<br>
-        <strong>Note:</strong> Data from the last six months are not final. The most recent data available in each visualization is for {{ thru_month_formatted }}.
+    <p class="m-chart-footnote block__sub block__border-top block short-desc" style="font-size:14px !important">
+    <strong>Source:</strong> National Mortgage Database<br>
+    <strong>Date published:</strong> {{ pub_date_formatted }}<br>
+    <strong>Downloads:</strong> CSV files with data by <a class="icon-link icon-link__download" href="{{ state_meta['url'] }}" <span class="icon-link_text">state</span></a> ({{ state_meta['size'] }}), <a class="icon-link icon-link__download" href="{{ metro_meta['url'] }}" <span class="icon-link_text">metro and non-metro areas</span></a> ({{ metro_meta['size'] }}), or <a class="icon-link icon-link__download" href="{{ county_meta['url'] }}"><span class="icon-link_text">county</span></a> ({{ county_meta['size'] }}).<br>
+    <strong>Notes:</strong> {{ viz_notes['map_notes'] }}<br>
```

Comment:
No lorem ipsum? 😉

Reply:
"Lorem ipsum" clearer than "Map notes to come." ??

Reply:
No real argument from me on this, or change request. Just following orders and looking at the mockup. 😆